File: src/DemoBattle/StraightFsApi/HeroController.fs
namespace StraightFsApi

open Common
open Types
open Newtonsoft.Json
open System.IO
open System.Web.Http

type HeroController() =
    inherit ApiController()

    member this.Get () =
        JsonConvert.DeserializeObject<FighterRepresentation[]>(File.ReadAllText(CommonStrings.HeroStoragePath))

File: src/DemoBattle/StraightCsApi/HeroController.cs
using System.Collections.Generic;
using System.Web.Http;
using System.Web.Http.Cors;
using System.IO;
using Common;
using Newtonsoft.Json;

namespace StraightCsApi
{
    [Route("api/hero")]
    [EnableCors(origins: "*", headers: "*", methods: "*")]
    public class HeroController : ApiController
    {
        public IEnumerable<Fighter> Get() =>
            JsonConvert.DeserializeObject<Fighter[]>(File.ReadAllText(CommonStrings.HeroStoragePath));
    }
}
